internal/common/baseclient/basepresident.go

```diff
@@ -58,7 +58,8 @@ func (p *BasePresident) SetTaxationAmount(islandsResources map[shared.ClientID]s
 
 	for clientID, clientReport := range islandsResources {
 		if clientReport.Reported {
-			taxAmountMap[clientID] = shared.Resources(float64(clientReport.ReportedAmount) * rand.Float64())
+			taxRate := 0.1
```

Comment:
is this a constant?

Reply:
can it be changed?

Reply:
ah nvm, just relaised this is baseclient.

Reply:
that is for basepresident, which will be overwritten by client implementation. Because basepresident just shows what the functions can do, it might as well be a constant